tests/testthat/test-generate_data.R

```diff
@@ -154,8 +153,7 @@ test_that("ordinal categorical data is generated correctly.", {
 
 
 test_that("non-proportional ordinal categorical data are generated correctly.", {
-  oldSeed <- .Random.seed
-  set.seed(87920)
+  skip_on_cran()
```

Comment:
You are skipping all tests on cran now? Where there issues last submission?

Reply:
It was a while ago - I don't remember exactly why I started doing this. I think there was an issue, and this was a simple work around. Do you think it is bad practice?

Reply:
Well it kind of subverts the testing process on CRAN... but you are certainly not the only one doing it so :shrug:

Reply:
I'll keep them in for now, and can revisit later.